### DISCUSSION TITLE
Per User Model Whitelisting

### DISCUSSION THREAD

**Discussion by davidakerr** [open-webui/open-webui#1536] [Ideas]
It would be very useful to limit certain users to certain models.

![image](https://github.com/open-webui/open-webui/assets/31540828/54812838-1dff-4248-b720-0e1bf215c89d)

**Comment by tjbck** [comment]
Related: #1450

**Comment by rkconsulting** [comment]
would love this capability as well!

**Comment by SyndicatedPillbug** [comment]
This would be amazing, yes, please!
